src/blog/lagom-1-5-0-RC1.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Lagom 1.5.0-RC1 released!
+date: 2018-11-16
+author_github: renatocaval
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.5.0-RC1
+---
+
+We are happy to announce the availability of Lagom 1.5.0-RC1. This is the first release candidate of Lagom 1.5.0. We expect this to be considerably more stable than the milestone releases, not only in terms of functionality but we are also moving closer to the idea of freezing the APIs.
+
+As with milestones, the primary goal is to get feedback, so please let us know if something isn't working or you see something that should be improved. If there are changes not well documented in javadocs, scaladocs or our migration guides, please, let us know so that we can improve them before the general availability release.
+
+
+## Highlights
+
+### Upgrades for Play, Akka and Alpakka Kafka
+
+- Play upgraded to 2.7.0-RC3 ([release notes](https://blog.playframework.com/play-2-7-0-rc3-released/)) 
+- Akka upgraded to 2.5.18 ([release notes](https://akka.io/blog/news/2018/10/07/akka-2.5.18-released))
+- Alpakka Kafka upgraded to 1.0-M1 ([release notes](https://doc.akka.io/docs/akka-stream-kafka/current/release-notes/1.0-M1.html))
+
+See GitHub for the [full list of commits](https://github.com/lagom/lagom/compare/1.5.0-M4...1.5.0-RC1) and the issues and pull requests assigned to the [1.5.0-RC1 milestone](https://github.com/lagom/lagom/milestone/39?closed=1).
+
+## Updating a Lagom project to version 1.5.0-RC1
+
+To update an sbt project, change the version of `lagom-sbt-plugin` in your `project/plugins.sbt` file.
```

Comment:
Should we link to the migration guide? So that users coming from 1.4.x can better understand how to try RC1.